src/logic/operate.js

```diff
@@ -0,0 +1,22 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  if (operation === '+') {
+    return Big(numberOne) * 1 + Big(numberTwo) * 1;
+  }
+  if (operation === '-') {
+    return Big(numberOne) - Big(numberTwo);
+  }
+  if (operation === 'x') {
+    return Big(numberOne) * Big(numberTwo);
+  }
+  if (operation === '÷') {
+    return Big(numberOne) / Big(numberTwo);
+  }
+  if (operation === '%') {
+    return Big(numberOne) * 0.01;
+  }
+  return '0';
+};
```

Comment:
- [ ] Please use Big.js mathematical operations here instead. Eg: `Big(numberOne).plus(Big(numberTwo))`.

- [ ] Also, to simplify things simply call a new variable at the top and wrap both numbers in `Big.js` rather than always wrapping them in `Big.js` at the point of each operation.

Reply:
I have implemented the changes suggested here